Makefile

```diff
@@ -223,9 +224,14 @@ VM_OPERATOR_IMAGE_NAME ?= extra/vm-operator
 VM_OPERATOR_CONTROLLER_IMG ?= $(STAGING_REGISTRY)/$(VM_OPERATOR_IMAGE_NAME)
 VM_OPERATOR_DIR := test/infrastructure/vm-operator
 VM_OPERATOR_TMP_DIR ?= vm-operator.tmp
-VM_OPERATOR_VERSION ?= v1.8.1
+VM_OPERATOR_COMMIT ?= de75746a9505ef3161172d99b735d6593c54f0c5
```

Comment:
Let's add a comment what that commit is. If it's not a specific tag, let's add something like " # commit from main from 17.04.2024"

(this makes it easier for us going forward to figure what how outdated it is / when to update. Because we don't have to do that lookup manually)

Reply:
Ups. Could have seen that it's the same id as v1.8.6-0-gde75746a, anyway, comment doesn't hurt :)